packages/@webex/plugin-cc/test/unit/spec/services/task/TaskManager.ts

```diff
@@ -0,0 +1,296 @@
+import 'jsdom-global/register';
+import EventEmitter from 'events';
+import { LoginOption, WebexSDK } from '../../../../../src/types';
+import { CALL_EVENT_KEYS, CallingClientConfig, LINE_EVENTS } from '@webex/calling';
+import { CC_EVENTS } from '../../../../../src/services/config/types';
+import TaskManager from '../../../../../src/services/task/TaskManager';
+import * as contact from '../../../../../src/services/task/contact'
+import Task from '../../../../../src/services/task';
+import { TASK_EVENTS } from '../../../../../src/services/task/types';
+import WebCallingService from '../../../../../src/services/WebCallingService';
+import config from '../../../../../src/config';
+
+jest.mock('./../../../../../src/services/task', () => {
+  return jest.fn().mockImplementation(() => {
+    return {
+        updateTaskData: jest.fn().mockReturnThis(),
+        emit: jest.fn()
+    };
+  });
+});
+
+describe('TaskManager', () => {
```

Comment:
Can we add UT for Agent wrapup event added where task is removed from the collection and also where end has happened from the customer end and taks: end is emitted

Reply:
Added them.